server/src/handlers/event.rs

```diff
@@ -74,7 +80,16 @@ pub async fn query(_req: HttpRequest, json: web::Json<Value>) -> HttpResponse {
 
 pub async fn post_event(req: HttpRequest, body: web::Json<serde_json::Value>) -> HttpResponse {
     let stream_name: String = req.match_info().get("logstream").unwrap().parse().unwrap();
-    let labels = utils::collect_labels(&req);
+
+    let tags = match collect_labelled(&req, PREFIX_TAGS, ',') {
+        Ok(tags) => HashMap::from([("p_tags".to_string(), tags)]),
+        Err(e) => return e.error_response(),
+    };
+
+    let metadata = match collect_labelled(&req, PREFIX_META, ',') {
+        Ok(metadata) => HashMap::from([("p_metadata".to_string(), metadata)]),
```

Comment:
Create a const with `,` as a value. Use that in passing as args, that would allow changing this value at a single point if needed